src/test/resources/view/UiPartTest/validFileWithFxRoot.fxml

```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
+<?import seedu.address.ui.TestFxmlObject?>
```

Comment:
What does this import do?

Reply:
not sure. but i found it in the team repo and it was missing from my own. so i just put it back there

Reply:
what's probably happening is that the class / object that is being made from this fxml file requires that import